.github/workflows/build.yml

```diff
@@ -1,5 +1,9 @@
+name: Build Kernel
 on:
   push:
+    branches:
+      - main
+      - maint-6.6
```

Comment:
this change would also need to be made in the maint-6.6 branch, having this in main will have no effects on actions on the other branch